pages/terms_of_service.md

```diff
@@ -116,6 +116,8 @@ When you use our Platform, you will be presented with a series of exercises for
 
 You agree that all content (**User Contributed Content**) that you have previously uploaded, or subsequently upload, to the Platform -- including, without limitation, your Solution and Iteration content -- will be licensed to Exercism on worldwide, royalty-free, non-exclusive, perpetual basis. You grant Exercism the perpetual and irrevocable right and license to process, copy, distribute and to commercially exploit all such User Contributed Content that you contribute. You should understand that this right will persist even if your User Contributed Content is subsequently removed by you OR if you subsequently delete your account on our Platform.
 
+When you delete your account, we erase all the personal details from your account. Your public solutions and comments will still remain the Platform but will be anonymised. This is an irreversal process, so please be sure before you delete your account.
```

Comment:
```suggestion
When you delete your account, we erase all the personal details from your account. This is an irreversal process, so please be sure before you delete your account. Your solutions and any comments made as a mentor will still remain the Platform but will be anonymised. 
```

I've reordered the sentences for clarity, and also edited the "what remains" sentence for clarity (mentor comments stay but general comments don't). 

@ccare If you merge this suggestion, I'm happy for the whole thing to be merged.

Reply:
I tweaked you suggestion a little @iHiD